src/Vue/Resources/assets/src/register_controller.ts

```diff
@@ -9,6 +9,12 @@
 
```

Comment:
There is another `(window as any).resolveVueComponent` in this file, can you update it plz?

Thanks !

Reply:
@Kocal, done.